docs/publications/BioHackrXiv/paper.md

```diff
@@ -0,0 +1,215 @@
+
+The workflow is run weekly via GitHub Actions continuous integration, providing an up-to-date table for each community. 
+The usage of an iframe enables updates for the table to propagate automatically to any website where it is deployed. 
+
+Any Galaxy community can use the pipeline by adding a folder in the [project GitHub repository](https://github.com/galaxyproject/galaxy_tool_extractor/). 
+To initialise the pipeline for a new community you need to add a new subfolder of `data/communities/` , and inside it add a file called 'categories' with a list of Galaxy ToolShed (**cite** doi:10.1186/gb4161 ) categories. 
+Additionally, tools that should be excluded or included after filtering, can be added to respective files as well. 
+A working example of the community configuration files can be found in the folder for the microgalaxy community.
+
+![Workflow of the Galaxy tool metadata extractor pipeline. Tool wrappers are parsed from different repositories and additional metadata is retrieved from bio.tools, Bioconda, and the main public Galaxy servers. Upon filtering and manual curation of the data for specific scientific communities, the data is transformed into interactive web tables and a tool usage statistics-based word cloud, that can be integrated into any website. \label{metadata_extractor_pipeline}](./figures/flowchart.png)
+
+
+## Annotation workflow
+
+The annotation process begins by selecting a tool from a Galaxy community. 
+This step can make use of the interactive table created by the Galaxy tool extractor scripts presented above. 
+A curator then needs to visit the development repository of the Galaxy tool wrapper and search the XML file for a bio.tools xref snippet (Figure \ref{xref_snippet}). 
+
+![xref snippet example for a Galaxy tool wrapper that contains the tool [Racon](https://bio.tools/Racon). \label{xref_snippet}](./xref.png)
+
+bio.tools is then checked to confirm that a bio.tools identifier does, or does not, exist. 
+The reason for this is that even if a bio.tools identifier exists in a tool wrapper, it may not necessarily exist in bio.tools. 
+This is an observation based on real-world annotation errors and serves as a useful supporting step to improve Galaxy wrapper annotations and the completeness of the bio.tools registry. 
+In addition, if a bio.tools identifier is not included in the wrapper, this does not mean that there is not a bio.tools identifier available in the registry.
+
+There are then two curation paths to choose from, depending on whether a bio.tools identifier exists in the XML wrapper. In both cases, if no bio.tools entry exists, a new entry should be created and updated using the bio.tools wizard. The creation and update of an entry includes adding concepts from the EDAM ontology. This annotation process can be simplified through the use of [EDAM Browser](https://edamontology.github.io/edam-browser/)[@citesAsAuthority:edamBrowser].
```

Comment:
```suggestion
There are then two curation paths to choose from, depending on whether a bio.tools identifier exists in the XML wrapper. In both cases, if no bio.tools entry exists, a new entry should be created and updated using the bio.tools wizard. The creation and update of an entry includes adding concepts from the EDAM ontology. This annotation process can be simplified through the use of [EDAM Browser](https://edamontology.github.io/edam-browser/)[@citesAsAuthority:edamBrowser, @citesAsAuthority:edamBrowserCode].
```
I don't insist we need this additional citation of the code, and I'm pesimistic it's counted anywhere, but I'm rather curious how it will look when trying to follow the "most proper" guidelines for citing scientific software 😅